ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -463,7 +463,12 @@ private class JdbcLedgerDao(
     contractsReader.lookupContractKey(forParty, key)
 
   private def splitOrThrow(id: EventId): NodeId =
-    split(id).fold(sys.error(s"Illegal format for event identifier $id"))(_.nodeId)
+    TransactionIdWithIndex
```

Comment:
I think this can be inlined too, the custom error message seems unnecessary (and probably misleading).

Reply:
done